react/src/components/DeleteMapModal/DeleteMapModal.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import { Modal, ModalHeader, ModalBody, ModalFooter } from 'reactstrap';
+import { Button, SectionMessage } from '@tacc/core-components';
+import { Project } from '../../types';
+import { useDeleteProject } from '../../hooks/projects/';
+
+type DeleteMapModalProps = {
+  isOpen: boolean;
+  close: () => void;
+  project: Project;
+};
+
+const DeleteMapModal = ({
+  isOpen,
+  close: parentToggle,
+  project,
+}: DeleteMapModalProps) => {
+  const {
+    mutate: deleteProject,
+    isLoading: isDeletingProject,
+    isError,
+    isSuccess,
+  } = useDeleteProject(project.id);
+  const handleClose = () => {
+    parentToggle();
+  };
+
+  const handleDeleteProject = () => {
+    deleteProject(undefined, {});
+  };
+
+  return (
+    <Modal size="lg" isOpen={isOpen} toggle={handleClose}>
+      <ModalHeader toggle={handleClose}>
+        Delete Map: {project?.name}{' '}
+      </ModalHeader>
+      <ModalBody>
+        {project?.deletable ? (
+          <>
+            Are you sure you want to delete this map? All associated features,
+            metadata, and saved files will be deleted.
+            {project?.public && <b> Note that this is a public map. </b>}
+            <br />
+            <b>
+              <u>This cannot be undone.</u>
+            </b>
+          </>
+        ) : (
+          "This map cannot be deleted because you don't have correct permission credentials."
```

Comment:
maybe "You don’t have permission to delete this map."

I see the point that they don't know why the button is disabled on project listing. I'm not certain the best UX for that like is it better to get them to click on something to then see they can't do the action.  could also add hover text for the disabled button but they probably won't see that either 🤔 .